src/Tasks/AssemblyDependency/ReferenceTable.cs

```diff
@@ -2687,6 +2687,17 @@ internal void GetReferenceItems
             copyLocalFiles = copyLocalItems.ToArray();
         }
 
+        internal static ITaskItem[] NormalizeAsArray(List<ITaskItem> items)
```

Comment:
```suggestion
        internal static ITaskItem[] NormalizePathsAsArray(List<ITaskItem> items)
```

How about this?

Reply:
I don't know if you saw, but I took the middle suggestion. I do want to mention "Array," since otherwise it would be very non-obvious (without looking at its return type) that it's returning an array instead of just normalizing it.